### PR TITLE
Update landing page

### LINKED ISSUE #420: define a skin for INGC
work with Jose and other stakeholders to establish a look and feel for 
- http://geonode.ingc.gov.mz and
- http://geonode.kartoza.com

@lucernae can you retrieve the skin we did a couple of years ago for http://geonode.inasafe.org/ and apply it there?

then implement if you can or hand over to a dev that can

depends on https://github.com/kartoza/docker-geosafe/issues/120  for deployment but design can proceed already

child of #458

### PR DESCRIPTION
**Pull request by boney-bun**
fix kartoza/geonode#420

### CODE REVIEW

File: templates/geosafe/analysis/create.html [hunk 1]
```diff
@@ -133,33 +133,33 @@
         }
 
         .option .set .badge {
-            background-color: #50cf5c;
+            background-color: #1a6c38;
         }
 
         .option .has-children:not(.set) .badge {
-            background-color: #3db2dd;
+            background-color: #2aa858;
```
Comment (myarjunar) [inline templates/geosafe/analysis/create.html:140]: if we decided to set the color for not selected layer to green, we need to change the color for selected layer (it's also green) as it's not obvious for the user which one means what IMO. cc @carina1000

File: templates/geosafe/analysis/create.html [hunk 2]
```diff
@@ -133,33 +133,33 @@
         }
 
         .option .set .badge {
-            background-color: #50cf5c;
+            background-color: #1a6c38;
```
Comment (myarjunar) [inline templates/geosafe/analysis/create.html:136]: making it darker green did not give a significant difference between not selected and selected layer IMO. cc @carina1000

**Comment by lucernae** [comment]
@boney-bun can we close it now? Wasn't it for INGC skin?

**Comment by boney-bun** [comment]
@lucernae yes, we can close this and focus on the INGC theme instead.
thanks

**Pull request closed without merging**